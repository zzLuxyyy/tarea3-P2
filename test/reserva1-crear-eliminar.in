crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1966
crearLibro 411222335 "Todos los fuegos el fuego" "Julio" "Cortázar" "Ocho cuentos que combinan lo cotidiano con lo fantástico" 1
crearReserva
imprimirReserva
liberarReserva
crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1968
crearLibro 431244557 "Lo bello y lo triste" "Yasunari" "Kawabata" "Un escritor se reencuentra con una amante del pasado en una historia de dolor y deseo" 1
crearReserva
imprimirReserva
liberarReserva
Fin