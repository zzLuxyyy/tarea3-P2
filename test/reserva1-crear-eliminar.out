1>Socio de ci 12345678 creado exitosamente.
2>3>4>Reserva de libro de isbn 411222335 a socio 12345678 creada exitosamente.
5>Reserva de libro Todos los fuegos el fuego por Jim Dear.
6>Se ejecutó 'liberarTReserva' exitosamente.
7>Socio de ci 12345678 creado exitosamente.
8>9>10>Reserva de libro de isbn 431244557 a socio 12345678 creada exitosamente.
11>Reserva de libro Lo bello y lo triste por Jim Dear.
12>Se ejecutó 'liberarTReserva' exitosamente.
13>Fin.
